usr/bin/nxdk-configure

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+. ${NXDK_DIR}/usr/share/nxdk/prelude.inl
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+# Wrapper around autoconf ./configure scripts
```

Comment:
I had some trouble with SDL2_gfx in test/.
I believe I had to `autoconf` or `autoreconf` before this would build without finding host libs.

That said, the SDL2_gfx README does tell the user to update their scripts.